src/api/telemetry/TelemetryAPI.js

```diff
@@ -142,6 +144,17 @@ define([
         this.valueFormatterCache = new WeakMap();
     }
 
+    /**
+     * Return Custom String Formatter
+     *
+     * @param {Object} metadata
+     * @param {Object} item
```

Comment:
I'm confused about what `item` is here? This is public API, it needs to be clear how it's used.